packages/modular-scripts/src/buildPackage/makeBundle.ts

```diff
@@ -44,49 +44,45 @@ export async function makeBundle(
   const { main, compilingBin } = getPackageEntryPoints(packagePath);
 
   if (!packageJson) {
-    throw new Error(
-      `no package.json in ${packagesRoot}/${packagePath}, bailing...`,
-    );
+    throw new Error(`no package.json in ${packagePath}, bailing...`);
   }
   if (packageJson.private === true) {
-    throw new Error(
-      `${packagesRoot}/${packagePath} is marked private, bailing...`,
-    );
+    throw new Error(`${packagePath} is marked private, bailing...`);
   }
 
-  if (!fse.existsSync(path.join(packagesRoot, packagePath, main))) {
+  if (!fse.existsSync(path.join(getModularRoot(), packagePath, main))) {
```

Comment:
You call `getModularRoot` 6 times in this file. Why not just declare it at the top?

Reply:
I guess with autocomplete this is just what I type because TSServer infers the call. Given it's memoized it's 0 cost, but I can see how it looks a bit messy.